tests/background_cpu_test.py

```diff
@@ -12,7 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-"""Tests for background workload"""
+"""Tests for background cpu workload & background workload framework"""
```

Comment:
No, not exactly, the first couple of tests here are not CPU workload specific.  The `testBackgroundWorkloadSpec` doesn't test the background CPU specifically, just that the prepare/start/stop functions are called.